jetson/nav/obstacle_avoidance/obstacleAvoidanceStateMachine.cpp

```diff
@@ -68,7 +68,9 @@ NavState ObstacleAvoidanceStateMachine::run()
 bool ObstacleAvoidanceStateMachine::isTargetDetected ()
 {
     return ( mRover->roverStatus().currentState() == NavState::SearchTurnAroundObs &&
```

Comment:
Are you checking that you are in a turn around obstacle state here because you're assuming that you'll only be able to see the target if it is being detected as an obstacle?

Reply:
No, this is a state for checking that if we happen to already be turning around an obstacle, but we also happen to see the target in the view while turning (this would make it so that instead of us going around the obstacle and resuming a search, if we are able to locate it while turning around the obstacle, we can just turn to that target and drive to it)

Reply:
That makes sense. Thanks